aladdin.sh

```diff
@@ -52,6 +53,14 @@ function get_plugin_dir() {
     fi
 }
 
+function get_install_software() {
```

Comment:
`aladdin config set install_software false` once is all we need to skip software installation for all future invocations of aladdin.